vpr/src/base/clustered_netlist.h

```diff
@@ -228,6 +228,21 @@ class ClusteredNetlist : public Netlist<ClusterBlockId, ClusterPortId, ClusterPi
     ///@brief Sets the flag in net_is_global_ = state
     void set_net_is_global(ClusterNetId net_id, bool state);
 
+    /**
+     * @brief Given a name of a block and vector of possible cluster blocks
+     *        that are candidates to match the block name, go through 
+     *        the vector of cluster blocks and return the id of the block
```

Comment:
Return first?
What is none matches (return Invalid?)

Reply:
Also seems like the comment is half-repeated?

Reply:
Done